src/v1.js

```diff
@@ -16,7 +16,7 @@ let _lastNSecs = 0;
 // See https://github.com/uuidjs/uuid for API details
 function v1(options, buf, offset) {
   let i = (buf && offset) || 0;
-  const b = buf || [];
+  const b = buf || new Array(16);
```

Comment:
In `v4` we now use a `Uint8Array(16)`, why not here?
https://github.com/uuidjs/uuid/blob/a361e29a68bbad78318715e6aee696f0006f4b34/src/v4.js#L6

Reply:
@ctavan typed array does not increase performance (in this case) =(
Performance remains at the level with an empty array `[]`.

Reply:
That performance implication is really interesting/unexpected. [Stackoverflow says](https://stackoverflow.com/a/24855023/1053532):

> Modern engines will use true arrays behind the scenes even if you use Array if they think they can, falling back on property map "arrays" if you do something that makes them think they can't use a true array.

However I just realized that this line in `v4` was deprecated (and never used) anyways, so please ignore my initial comment. See #454

Let's simply leave `new Array(16)` in place. Pre-allocating the size of the array definitely is a reasonable optimization.